src/components/OsmContribution.jsx

```diff
@@ -5,7 +5,7 @@ import Telemetry from 'src/libs/telemetry';
 const OsmContribution = ({ poi }) => {
   const [_osmKey, itemKind, itemId] = poi.id.split(':');
   const viewUrl = `https://www.openstreetmap.org/${itemKind}/${itemId}`;
-  const editUrl = `https://www.openstreetmap.org/edit?editor=id&${itemKind}=${itemId}`;
+  const editUrl = `https://www.openstreetmap.org/edit?editor=id&${itemKind}=${itemId}&hashtags=QwantMaps`;
```

Comment:
Oups, line too long for the linter :)
You could use this util function if you want https://github.com/QwantResearch/erdapfel/blob/master/src/libs/url_utils.js#L64

Reply:
Indeed. I didn't go for the helper: as one of the keys values is defined by a variable (`itemKind`), it may be slightly less concise to define a dictionary with query parameters. Unless I miss a Javascript feature or just some coding convention.

Reply:
If you mean dynamic object key names, you can do it indeed, by using [ ] :)
```js
{
   editor: 'id',
   [itemKind]: 'itemId',
}
```